doc/apps/dash.md

```diff
@@ -104,6 +109,27 @@ ploomber-cloud deploy --watch
 To ensure your app doesn't break on re-deployments, pin your [dependencies.](pin-dependencies)
 ```
 
+(customize-deployment)=
+## Customize deployment
+
+To customize the Python version or base image for deployment, you may use [Docker deployment](./docker.md).
```

Comment:
you can create a sample project for each framework: `examples/{framework}/docker-based` and put the dockerfile, along with a simple app there.